roofit/multiprocess/inc/RooFit/MultiProcess/Config.h

```diff
@@ -13,13 +13,30 @@
 #ifndef ROOT_ROOFIT_MultiProcess_Config
 #define ROOT_ROOFIT_MultiProcess_Config
 
+#include <cstddef>  // std::size_t
+
 namespace RooFit {
 namespace MultiProcess {
 
 class Config {
 public:
    static void setDefaultNWorkers(unsigned int N_workers);
    static unsigned int getDefaultNWorkers();
+
+   class LikelihoodJob {
+   public:
+      // magic values to indicate that the number of tasks will be set automatically
+      constexpr static std::size_t automaticNEventTasks = 0;
+      constexpr static std::size_t automaticNComponentTasks = 0;
+
+      static void setDefaultNEventTasks(std::size_t NEventTasks);
+      static std::size_t getDefaultNEventTasks();
+      static void setDefaultNComponentTasks(std::size_t NComponentTasks);
+      static std::size_t getDefaultNComponentTasks();
+   private:
+      static std::size_t defaultNEventTasks_;
+      static std::size_t defaultNComponentTasks_;
```

Comment:
```suggestion
      static std::size_t defaultNEventTasks;
      static std::size_t defaultNComponentTasks;
```
If the user can set and get the member values how they want anyway, it doesn't make that much sense to have getters and setters that trivially change the values. It would be less verbose to just have public members in the class that the caller can just modify.

Reply:
You are completely right, thanks!